netbout-web/src/test/casperjs/inbox-noresults-message.js

```diff
@@ -0,0 +1,32 @@
+/*globals casper:false */
+casper.test.begin(
+    'inbox can list bouts',
+    function (test) {
+        casper.start().then(
+            function() {
+                this.open(
+                    casper.cli.get('home') + "/search?q=invalidsearch",
+                    {
+                        method: 'GET',
+                        headers: {
```

Comment:
@mkordas Other tests provide it as well. It's one more validation, to check if netbout returns apropriate content :D